tasks/cmdrun/crtbuilder/crtbuilder.go

```diff
@@ -0,0 +1,54 @@
+package crtbuilder
```

Comment:
Not sure that packages for each task are that big that it would make sense to have a separate package for the builder as well. I would just put the builder in the task package.

Reply:
this is difficult due to circular dependencies, which is the reason that the builder in a separate package. we can discuss in teams chat maybe, but at the moment, i'm thinking that sorting out the dependencies further needs to be a subsequent refactoring.